packages/legacy-workbench/src/js/angular/guides/steps/complex/ttyg/ask-ttyg-agent/plugin.js

```diff
@@ -1,30 +1,28 @@
+const TTYG_ASK_DEFAULT_TITLE = 'guide.step-action.ask-ttyg-agent';
+
 PluginRegistry.add('guide.step', [
     {
-        guideBlockName: 'ask-ttyg-agent',
+        guideBlockName: 'ttyg-ask-question',
         getSteps: (options, services) => {
             const GuideUtils = services.GuideUtils;
-            options.mainAction = 'ask-ttyg-agent';
-            options.maxWaitTime = 10;
-            options.forceReload = false;
-
-            const explain = options.explain;
-            const explainMore = options.explainMore;
-            const exploreSparql = options.exploreSparql;
-
-            const steps = [
+            return [
                 {
                     guideBlockName: 'input-element',
                     options: angular.extend({}, {
                         skipPoint: true,
```

Comment:
Move the skipPoint option into the complex step. It is strange to see a 'Skip' button if we use this step alone.